### PR TITLE
Jdbc/sub branch

### PR DESCRIPTION
**Pull request by ykgautam**

### CODE REVIEW

File: java-examples/src/main/java/edu/practice/hashequals/Human.java [hunk 1]
```diff
@@ -1,15 +1,13 @@
 package edu.practice.hashequals;
-
 public class Human {
+	protected int adharNumber ;
```
Comment (Ganeshgautam) [inline java-examples/src/main/java/edu/practice/hashequals/Human.java]: make it private

File: java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSort.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package edu.practice.algos.sort.impl;
+
```
Comment (Ganeshgautam) [inline java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSort.java:2]: add javadocs on what this class does like:
/**
* This class sorts an array.
*
*/

Add this to all classes.

File: java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package edu.practice.algos.sort.impl;
+
+public class MergeSorting {
```
Comment (Ganeshgautam) [inline java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java:3]: The name is confusing... what is the difference between MergeSort class and MergeSorting class?

**Review by Ganeshgautam — CHANGES_REQUESTED** [review]

File: java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSort.java (hunk 2 above)
Comment (ykgautam) [reply java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSort.java:2]: kk

File: java-examples/src/main/java/edu/practice/hashequals/Human.java (hunk 1 above)
Comment (ykgautam) [reply java-examples/src/main/java/edu/practice/hashequals/Human.java]: done

File: java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java (hunk 3 above)
Comment (ykgautam) [reply java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java:3]: i will take care of this

**Pull request closed without merging**